QUEST.tsv

```diff
@@ -1649,151 +1649,154 @@ QUEST_20150323_001648	Huh?
-QUEST_20150323_001692	Disaster of Saknis Plains
-QUEST_20150323_001693	Objective of the Monsters of Vilna Forest
-QUEST_20150323_001694	Vilna Forest Altar
-QUEST_20150323_001695	Threat of Poslinkis Forest
-QUEST_20150323_001696	Food Pile of Big Starving Demon
+QUEST_20150323_001685	$Srautas Canyon is filled with evil forces and the demons have begun to attack. {nl}Guards and Paladin followers are defending against them but they desperately need your help.
+QUEST_20150323_001686	$Strange things are happening in Vejo Ravine. {nl}All the villagers disappeared and monsters took their place.
+QUEST_20150323_001687	$Monsters entered the Stele Road and are blocking the way. {nl}All towns and villages will be isolated if we don't get rid of the monsters.
+QUEST_20150323_001688	$We need support to make our way from the Dvaisa Ridge to Gateway of Great King. {nl}Please help.
+QUEST_20150323_001689	$The Gambler of Klaipeda
+QUEST_20150323_001690	
+QUEST_20150323_001691	$Investigate the Miners' Village
+QUEST_20150323_001692	$Disaster of Saknies Plains
+QUEST_20150323_001693	$Objective of the Monsters of Vilnios Forest
+QUEST_20150323_001694	$Vilnios Forest Altar
+QUEST_20150323_001695	$Threat of Poslinkio Forest
+QUEST_20150323_001696	$Food Pile of Starving Demon's Way
 QUEST_20150323_001697	For the Pilgrims
 QUEST_20150323_001698	True Nature of the Curse
-QUEST_20150323_001699	Landfill work
-QUEST_20150323_001700	Tough life
+QUEST_20150323_001699	$Landfill Work
+QUEST_20150323_001700	$Tough Life
 QUEST_20150323_001701	Need Health Tonic
-QUEST_20150323_001702	I want to eat normal food
-QUEST_20150323_001703	Pass along
-QUEST_20150323_001704	Handle the strange force
-QUEST_20150323_001705	Energy replenishment
+QUEST_20150323_001702	$I Want to Eat Normal Food
+QUEST_20150323_001703	$Pass Along
+QUEST_20150323_001704	$Handle the Strange Force
+QUEST_20150323_001705	$Energy Replenishment
 QUEST_20150323_001706	Positive Evidence
 QUEST_20150323_001707	Weapon Supply
 QUEST_20150323_001708	Food Reserve
-QUEST_20150323_001709	How to smash
-QUEST_20150323_001710	Monster gone wild
+QUEST_20150323_001709	$How to Smash
+QUEST_20150323_001710	$Monster Gone Wild
 QUEST_20150323_001711	Suspicious Sword
-QUEST_20150323_001712	Monster preying on military supplies
+QUEST_20150323_001712	$Monster Preying on Military Supplies
 QUEST_20150323_001713	Louis' Farmland
-QUEST_20150323_001714	Unsatisfactory crops
-QUEST_20150323_001715	Old seeds
-QUEST_20150323_001716	Shaton's Stubborness
+QUEST_20150323_001714	$Unsatisfactory Crops
+QUEST_20150323_001715	$Old Seeds
+QUEST_20150323_001716	$Shaton's Stubbornness
 QUEST_20150401_001717	Star Statue
 QUEST_20150401_001718	Nothing is happening…
-QUEST_20150401_001719	Supressing Altar
-QUEST_20150401_001720	Altar is already activated
-QUEST_20150401_001721	The candlestick is already lit
-QUEST_20150401_001722	Need something to light it…
+QUEST_20150401_001719	$Suppressing Altar
+QUEST_20150401_001720	$The altar is already activated.
```

Comment:
If it's a dialogue between 2 characters, "this" would fit, but it seems to be an interaction with the altar, so "The altar" would fit in that context.